addons/event_sale/views/product_views.xml

```diff
@@ -5,7 +5,7 @@
              <field name="model">product.template</field>
              <field name="inherit_id" ref="product.product_template_form_view" />
              <field name="arch" type="xml">
-                <group name="email_template_and_project" position="after">
+                <group name="email_template_and_project" position="before">
```

Comment:
Hello @AntoineVDV 

This PR is for stable version (13.0).

If we change the name `email_template_and_project` to `email_template `
and move the declaration of the group in `product_template_email/product_view.xml` this file.

Diff will increase.

also there is no meaning to keep sale group empty in  `product/views/product_views.xml`.

If you still need to change let me know i'll change according to your suggestion.

Thank you

Reply:
> This PR is for stable version (13.0)

After discussion with @dbo-odoo, it's better to do not anything about this issue (the white hole) in stable. The hole is there and stays there in 13.0.
Nonetheless, you can still fix it in the forward-port to master. Just don't forget to mention in the new commit message for 13.1 ;)

> also there is no meaning to keep sale group empty in product/views/product_views.xml.

Currently no but it will be necessary to do so after your changes since the views `sale.product_template_form_view_invoice_policy`, `sale_subscription.product_template_view_form_recurring` and `event_sale.event_sale_product_template_form` will extend the group `sale`. (Maybe some others, to check)

Reply:
I created a subtask for this case so no need to take care of it in the forward-port.

Reply:
> I created a subtask for this case so no need to take care of it in the forward-port.

ok